public/app/plugins/datasource/cloudwatch/tracking.ts

```diff
@@ -0,0 +1,93 @@
+import { DashboardLoadedEvent } from '@grafana/data';
+import { reportInteraction } from '@grafana/runtime';
+
+import { isCloudWatchLogsQuery, isCloudWatchMetricsQuery, isCloudWatchQuery } from './guards';
+import pluginJson from './plugin.json';
+import { CloudWatchMetricsQuery, MetricEditorMode, MetricQueryType } from './types';
+
+interface CloudWatchOnDashboardLoadedTrackingEvent {
+  grafana_version?: string;
+  dashboard_id?: string;
+  org_id?: number;
+  user_id?: number;
```

Comment:
I have found that the user_id is not very helpful and sometimes has the email in it, so seemed pretty sensitive. RS has an anonymous_id prop that seems better to count users, but I am not yet sure whether that's per session or per user 🤔